src/main/scala/org/broadinstitute/dsde/firecloud/utils/TSVParser.scala

```diff
@@ -10,16 +10,16 @@ package org.broadinstitute.dsde.firecloud.utils
 case class TSVLoadFile(
   firstColumnHeader:String, //The first header column, used to determine the type of entities being imported
   headers:Seq[String], //All the headers
-  tsvData:Seq[Map[String, String]] //List of rows of the TSV. Each row is a map from header to value.
+  tsvData:Seq[Seq[String]] //List of rows of the TSV, broken out into fields
 )
```

Comment:
I understand 2; for 1 you're saying "and it didn't even give me all the information I needed anyway so no great loss"?